incubator/hnc/pkg/controllers/object_controller_test.go

```diff
@@ -11,18 +11,25 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 
 	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/config"
 )
 
-var _ = Describe("Secret", func() {
+var _ = Describe("Role", func() {
```

Comment:
Didn't we switch this back to "Secret" now?

Reply:
Good catch! Switched back to "Secret".